homeassistant/components/pyload/sensor.py

```diff
@@ -167,7 +171,7 @@ async def async_update(self) -> None:
             self._attr_available = False
             return
         else:
-            self.data = status.to_dict()
+            self.data = status
```

Comment:
This seems like a breaking change in the library to return a `TypedDict` instead of a `dataclass`

I would have expected the major version to check if the library uses semver

Reply:
While I realize you are converting this to a dict anyways, if I was consuming the API, I probably would have preferred the dataclass even though it is a bit slower

Reply:
I agree with you, in hindsight i should have incremented the major version. Will try to remember it the next time.